src/lib/data/createables.ts

```diff
@@ -2501,6 +2501,14 @@ const Createables: Createable[] = [
 		outputItems: new Bank().add('Amulet of rancour').freeze(),
 		noCl: true
 	},
+	{
+		name: 'Zombie axe',
+		inputItems: new Bank().add('broken zombie axe').freeze(),
```

Comment:
Item names should always follow the osrs convention of `Broken zombie axe`, or (if the item is in it), use `EItem.ITEM_NAME`. I will merge without requesting a change because its not a big deal.